pincer/client.py

```diff
@@ -547,6 +551,63 @@ async def payload_event_handler(self, _, payload: GatewayDispatch):
         """
         await self.process_event("payload", payload)
 
+    async def wait_for(
+        self,
+        event_name: str,
+        check: Optional[Callable[[Any], bool]] = None,
+        timeout: Optional[float] = None
```

Comment:
is timeout as a `float` necessary, like don't seems necessary to put a timeout of `.5`  instead of `1`...
float seems kinda overkill

Reply:
It is since it's in seconds. Also, it just passes through to `asyncio.wait_for()` that uses floats.